app/delete_feature/delete.test.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+var expect = require('chai').expect;
+var request = require('supertest');
+
+var app = require('../../app/');
+var Post = require('../post/Post');
+var database = require('../../lib/mongo/');
+
+
+describe('deleting', function (){
+  var seededPost;
+  beforeEach(function (done){
+    //and one tweet from each
+    var post = {
+      text : 'HelloWorld'
+    };
+    database.connect(function (){
+      Post.collection.insert(post, function (err, data){
+          seededPost = data.ops;
+          done();
+        });
+    });
+  });
+
+  //drop collections after tests
+  after(function (done){
+      Post.dropCollection(done);
+    });
+
+  //  adding hidden to a post
+  it('should add hidden to the post', function (done){
+      request(app)
+        .delete('/post/' + seededPost[0]._id)
+        .expect(302)
+        .end(function (err){
+          if (err) throw err;
+          request(app)
+            .get('/post/' + seededPost[0]._id)
```

Comment:
Can you also perform this get request before the delete, to test that it returns properly